src/main/scala/nl/knaw/dans/lib/logging/servlet/masked/request/MaskedCookie.scala

```diff
@@ -30,9 +30,8 @@ private[masked] trait MaskedCookie extends RequestLogExtensionBase {
   }
 
   private def formatCookieValue(value: String): String = {
-    val cookieName = value.replaceAll("=.*", "")
-    val cookieValue = value.replaceAll(".*=", "")
-    val maskedCookieValue = cookieValue.replaceAll("[^.=]", "*") // replace everything but dots
+    val Array(cookieName, cookieValue) = value.split("=", 2)
```

Comment:
Throws a scala.MatchError if some fool of a client adds a header named 'cookie' without an '=' in the value.

Perhaps the method name should become `formatCookieHeader`, add scala doc: `header name 'cookie' for requests, header name 'set-cookie' for responses`.

Reply:
You're way smarter than me! I reverted to your solution with the two `replaceAll`. Is way better, after all!